test/views/btn/BTNConfirmationViewSpec.scala

```diff
@@ -48,6 +48,7 @@ class BTNConfirmationViewSpec extends ViewSpecBase {
 
     "have a h2 heading" in {
       groupView.getElementsByTag("h2").text() must include("What happens next")
+      groupView.getElementsByTag("h2").get(0).hasClass("govuk-heading-m") mustEqual true
```

Comment:
```suggestion
      val h2Headings: Elements = groupView.getElementsByTag("h2")
      h2Headings.get(0).text() mustB "What happens next"
      h2Headings.get(0).hasClass("govuk-heading-m") mustBe true
```

Reducing HTML DOM traversals by calling `getElementsByTag()` once and remove `must include`.

Reply:
Yea I left that line as I wasn't directly touching it (knowing we have the other ticket coming up) but with your suggestion I will be so I'll change that one too